sdk/identity/Azure.Identity.Broker/tests/PopTokenAuthenticationPolicy.cs

```diff
@@ -4,24 +4,26 @@
 using System;
 using System.Net;
 using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
 
-namespace Azure.Core.Pipeline
+namespace Azure.Identity.Broker.Tests
 {
     /// <summary>
     /// A policy that supports <see href="https://learn.microsoft.com/entra/msal/dotnet/advanced/proof-of-possession-tokens">Proof of Possession (PoP) tokens</see>.
     /// </summary>
     public class PopTokenAuthenticationPolicy : HttpPipelinePolicy
```

Comment:
For this migration, `PopTokenAuthenticationPolicy` doesn't need to go to core as well?

Reply:
We don't need to decide that until we add support to a service SDK, so I decided to leave it as test code for now.